cli/src/cli/index.ts

```diff
@@ -331,6 +331,7 @@ export const runCli = async (): Promise<CliResults> => {
     if (project.authentication === "next-auth") packages.push("nextAuth");
     if (project.database === "prisma") packages.push("prisma");
     if (project.database === "drizzle") packages.push("drizzle");
+    packages.push("geist");
```

Comment:
Just add it in base/package.json instead

Reply:
fixed in latest commit